src/components/templates/PartyMap/components/PartyMapPreview/PartyMapPreview.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FC } from "react";
+import { PartyMapRoomData } from "types/PartyMapRoomData";
+
+interface PartyMapPreviewProps {
+  expanded: boolean;
+  backgroundImage?: string;
+  backgroundRatio: number;
+  avatarPosition: {
+    x: number;
+    y: number;
+  };
+  rooms: PartyMapRoomData[];
+  onIconClick: () => void;
+}
+
+const PREVIEW_WIDTH_PIXELS = 120;
+const PREVIEW_WIDTH_VH = 80;
```

Comment:
Should these be set in pixels/etc? Will this play nice with responsive design/etc?